tests/integration/longview/test_longview.py

```diff
@@ -0,0 +1,119 @@
+import time
+
+from tests.integration.helpers import (
+    assert_headers_in_lines,
+    delete_target_id,
+    exec_test_command,
+)
+
+BASE_CMD = ["linode-cli", "longview"]
+
+
+def test_create_longview_client():
+    new_label = str(time.time_ns()) + "label"
+    exec_test_command(
+        BASE_CMD
+        + [
+            "create",
+            "--label",
+            new_label,
+            "--text",
+            "--no-headers",
+        ]
+    )
+
+
+def test_longview_client_list():
+    res = (
+        exec_test_command(BASE_CMD + ["list", "--text", "--delimiter=,"])
+        .stdout.decode()
+        .rstrip()
+    )
+    lines = res.splitlines()
+
+    client_id = lines[1].split(",")[0]
+
+    headers = ["id", "label", "created"]
+    assert_headers_in_lines(headers, lines)
+    return client_id
```

Comment:
Can we use fixtures here instead of return? This will cause warning prints when running with pytest